services/blockchain-indexer/shared/indexer/blockchainIndex.js

```diff
@@ -186,18 +185,18 @@ const indexBlock = async job => {
 
 		await blocksDB.upsert(blockToIndex, dbTrx);
 		await commitDbTransaction(dbTrx);
-		logger.debug(`Committed MySQL transaction to index block at height ${height}`);
+		logger.debug(`Committed MySQL transaction to index block ${block.id} at height ${block.height}`);
 	} catch (error) {
 		await rollbackDbTransaction(dbTrx);
-		logger.debug(`Rolled back MySQL transaction to index block at height ${height}`);
+		logger.debug(`Rolled back MySQL transaction to index block ${block.id} at height ${block.height}`);
 
 		if (error.message.includes('ER_LOCK_DEADLOCK')) {
-			const errMessage = `Deadlock encountered while indexing block at height ${height}. Will retry later.`;
+			const errMessage = `Deadlock encountered while indexing block ${block.id} at height ${block.height}. Will retry later.`;
 			logger.warn(errMessage);
```

Comment:
can we can move this to top so that we log for every case?

Reply:
Clarified offline!